tasks/main.yml

```diff
@@ -20,5 +20,20 @@
   tags:
     - haproxy
 
+- name: Make sure service dropin directory is available
+  file:
+    path: /etc/systemd/system/marathon.service.d/
+    state: directory
+  when: ansible_service_mgr == 'systemd'
```

Comment:
If you will add 'if mesos_ssl_enabled' to both tasks that create ssl then should this should work as expected